.travis.yml

```diff
@@ -0,0 +1,3 @@
+language: node_js
+node_js:
+  - "5"
```

Comment:
Hmm, have to check with @samsontmr on what node version he is running the bot on.